### PR TITLE
Read from production db

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Remove test firestore configurations
Read from production db, log incoming data & remove test plots code

### CODE REVIEW

File: public/media_engagement/media_engagement.js [hunk 1]
```diff
@@ -34,335 +35,3 @@ firebase.auth().onAuthStateChanged(function (user) {
         window.location.replace('auth.html')
     }
 });
-
```
Comment (lukechurch) [inline public/media_engagement/media_engagement.js:37]: This PR appears to be deleting a lot of functionality as well as changing to production db. Can you confirm that is intended?

Comment (IsaackMwenda) [reply public/media_engagement/media_engagement.js:37]: Yes thats intention. The next pr covers the new dashboard design.

Comment (lukechurch) [reply public/media_engagement/media_engagement.js:37]: ok thanks, in general that should probably have gone into a separate PR as it's a different logical change

**Review by lukechurch — APPROVED** [review]

File: public/media_engagement/media_engagement.js (hunk 1 above)
Comment (IsaackMwenda) [reply public/media_engagement/media_engagement.js:37]: Great got it.

**Pull request merged**